core/testcasecontroller/algorithm/paradigm/lifelong_learning/lifelong_learning.py

```diff
@@ -82,7 +82,9 @@ def run(self):
         samples_transfer_ratio_info = self.system_metric_info.get(
             SystemMetricType.SAMPLES_TRANSFER_RATIO.value)
         mode = self.model_eval_config.get("model_metric").get("mode")
-        if mode != 'multi-inference':
+
+        # in this mode, the inference period is skipped to accelerate training speed
+        if mode == 'no-inference':
```

Comment:
Is this code for test not for formal running? If it is, please modify it.

Reply:
This code provide multiple modes for running, users could choose either test or formal running in "algorithm.yaml".